parm/aqm.rc

```diff
@@ -34,28 +34,10 @@ dy_tracer_map_start: 11
 
 ctm_aod: true
 ctm_pmdiag: true
+ctm_photdiag: false
+ctm_depvfile: false
 ctm_stdout: maintask
 
-#
-# Inline Canopy Effects
```

Comment:
@MichaelLueken @benkozi  Can you comment as to why these canopy settings are removed from aqm.rc?